configure

```diff
@@ -68,251 +68,7 @@ echo "$SWIG_PATH" > tensorflow/tools/swig/swig_path
 # Invoke python_config and set up symlinks to python includes
 (./util/python/python_config.sh --setup "$PYTHON_BIN_PATH";) || exit -1
 
-## Set up Cuda-related environment settings
-
-while [ "$TF_NEED_CUDA" == "" ]; do
```

Comment:
Working on fixing these two issues.

By the way, is there a reason why the `fetch` command doesn't create any Bazel output dir symlinks? Seems that it might be useful to create the `bazel-<workspace-name>` symlink at least so that we can look at the external repositories that were fetched.